nix/lib/purescript.nix

```diff
@@ -49,4 +50,8 @@ stdenv.mkDerivation {
   installPhase = ''
     mv dist $out
   '';
+
+  postInstall = ''
+    npm prune --production
```

Comment:
comment?

Reply:
Added a comment explaining what is going on and why it is where it is